log_outgoing_requests/log_requests.py

```diff
@@ -15,6 +17,18 @@ def hook_requests_logging(response, *args, **kwargs):
     logger.debug("Outgoing request", extra=extra)
 
 
+def hook_requests_exception(e, *args, **kwargs):
+    extra = {
+        "requested_at": timezone.now(),
```

Comment:
that seems overkill, the logging library itself already tracks a timestamp of when the log record is emitted. Surely there will not be delta's between this time that cause issues that you're looking at the requests of the wrong day?